File: app/src/main/java/com/example/woo/songstar/utils/AppUtils.java
package com.example.woo.songstar.utils;

import android.content.Context;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class AppUtils {

    public static Retrofit getRetrofit() {
        return new Retrofit.Builder()
                .baseUrl("https://itunes.apple.com/")
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }
}

File: app/src/main/java/com/example/woo/songstar/utils/AppUtils.kt
package com.example.woo.songstar.utils

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

class AppUtils {

    companion object {
        fun getRetrofitInstance(): Retrofit {
            return Retrofit.Builder()
                .baseUrl("https://itunes.apple.com/")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
        }
    }
}